js/iframe-coordinator/karma.conf.js

```diff
@@ -0,0 +1,26 @@
+let webpackConfig = require('./webpack.config');
+
+module.exports = function(config) {
+  config.set({
+    basePath: '',
+    frameworks: ['jasmine'],
```

Comment:
Are we ok with jasmine?

Reply:
I've never found a JS unit testing library that I particularly liked, so... yes? :stuck_out_tongue_winking_eye: